build.gradle

```diff
@@ -30,7 +30,7 @@ sourceCompatibility = 11
 targetCompatibility = 11
 
 ext {
-    cradleVersion       = '3.1.1-slf4j-transitive-deps-2295891613-SNAPSHOT'
+    cradleVersion       = '3.1.2'
```

Comment:
Does this version has changes with the slf4j transitive dependencies?

Reply:
no this version is just cradle branch version-3 with serialization bug fixes and tests added for it